Doc/library/os.rst

```diff
@@ -3780,6 +3780,174 @@ features:
    .. versionadded:: 3.10
 
 
+.. function:: timerfd_create(clockid, flags)
+
+   Create and return an timerfd file descriptor. The file descriptors supports
```

Comment:
Should not it be `a timer file descriptor`?

Reply:
@serhiy-storchaka 
Fixed.